<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductDialog</class>
 <widget class="QDialog" name="ProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Виробник</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ціна</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Кількість</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="shelfLifeLineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="quantityLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Назва</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Кількість калорій</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="manufacturerLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="priceLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Склад продукту</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="caloriesLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Термін зберігання</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTextEdit" name="ingredientsTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
